CRM/Utils/System.php

```diff
@@ -301,6 +301,16 @@ public static function url(
   }
 
   /**
+   * @return string|null
+   */
+  public static function getUrlPath() {
+    if (isset($_GET[CRM_Core_Config::singleton()->userFrameworkURLVar])) {
+      return $_GET[CRM_Core_Config::singleton()->userFrameworkURLVar];
+    }
+    return NULL;
+  }
```

Comment:
So in the spirit of our recent comments fix ups the comment should have a sentence description to meet the full drupal standard